src/servers/src/http/handler.rs

```diff
@@ -215,6 +215,7 @@ impl From<PromqlQuery> for PromQuery {
             start: query.start,
             end: query.end,
             step: query.step,
+            lookback: DEFAULT_LOOKBACK_STRING.to_string(),
```

Comment:
I wonder if it's easy to support lookback param in the HTTP endpoint also.

Generally it can be a follow-up, but I'd prefer to add it in this patch as long as it's trivial to convey.

Reply:
I think I did one attempt earlier to support it but end up with a lot of changes... so decided to keep it simple.
But if the current approach doesn't look too useful without http/gprc support, of course let's try adding it

Reply:
> but end up with a lot of changes

That can make sense. Since we don't support it now, could you open an issue describing your attempt and outline the challenges? Then we can postpone it a bit with certain progress you made :D